### PR TITLE
First draft for Git article

### PR DESCRIPTION
**Pull request by alxmrs**

### CODE REVIEW

File: src/drafts/2020-04-git.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## `$ git`
+
+Have you ever taken a sober look at the output of running `git` on its own? Welcome are the familiar `clone`, `init`, `commit`, `push` and `pull` --
+commands we know and love.
+ 
+I never knew that git had tutorial documentation. What are these `git help tutorial`, `git help everyday`, `git help revisions` and `git help workflows`?
+
+I've been meaning to investigate `bisect` sometime. I've heard it lets you binary search commits to track down when a bug was introduced. 
+`rebase` catches my attention. I *think* I know how to rebase, but can one every *really* know?
```
Comment (ships) [inline src/drafts/2020-04-git.md:21]: every -> ever

File: src/drafts/2020-04-git.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Apparent to me after a while is the total absence of `checkout`. I use that command everyday. Yesterday, I discovered via total guess that I can jump back and forth between two branches with `git checkout -`. 
+Have I fallen behind the times? What is this new kid `switch` on the block? I hope `checkout` gets introduced later, if only to know the origins of the character that I so love.
```
Comment (ships) [inline src/drafts/2020-04-git.md:24]: this line excites me because it introduces something i didn't already know

File: src/drafts/2020-04-git.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+k
+## `$ git help -a`
+
+This is what I was looking for! Well, I didn't know what to expect here. But I wanted to see what I'm missing from git. It's a *lot*.
+I remember when a co-worker told me about `cherry-pick` and how big a deal that was to me. `cherry-pick` got me out of so many binds. 
+It remains to this day by default backup option when things get really messy.
```
Comment (ships) [inline src/drafts/2020-04-git.md:30]: by -> my

File: src/drafts/2020-04-git.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+I learned git in college. I would say that I learned it in a specific class, except around then, they were in the middle of
+changing the curriculum around. So they covered git in four classes. And all four classes covered basically the same content.
+
+Having used git everyday for several years, I realize that I have only really scratched the surface of what's possible. 
+Sure, I'll hear debates from my coworkers or articles on the best workflow for a given project. There, I'll learn a new 
+trick that I didn't know before. But this doesn't satisfy my craving to really *know* git -- to see all it can do.
+
+## `$ git`
+
+Have you ever taken a sober look at the output of running `git` on its own? Welcome are the familiar `clone`, `init`, `commit`, `push` and `pull` --
```
Comment (ships) [inline src/drafts/2020-04-git.md:15]: why use "sober" here? is it related "sobering thought" or "i'm sober" more?

File: src/drafts/2020-04-git.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+title: Notes from reading the git manpages
+date: 2020-04-29
+---
+
+I learned git in college. I would say that I learned it in a specific class, except around then, they were in the middle of
+changing the curriculum around. So they covered git in four classes. And all four classes covered basically the same content.
+
+Having used git everyday for several years, I realize that I have only really scratched the surface of what's possible. 
+Sure, I'll hear debates from my coworkers or articles on the best workflow for a given project. There, I'll learn a new 
+trick that I didn't know before. But this doesn't satisfy my craving to really *know* git -- to see all it can do.
```
Comment (ships) [inline src/drafts/2020-04-git.md:11]: I suspect this craving is actually your point -- can you introduce it more strongly

Comment (ships) [reply src/drafts/2020-04-git.md:11]: "there's so much to git that i don't understand" -> per our call. the key is why do you care? There's something super personal to this that i think you can bring out. In other words i don't think you're exploring an arbitrary curiosity about a complicated object -- you have stake in git somehow, a relationship with it.

It's almost a factional alignment that you are suggesting. Or something controversial. "Why don't I [ergo, why doesn't everyone] explore this more deeply? What is the *happening* of the me/git entanglement? You provoke feelings of curiosity when you name some less well known features. This suggests your audience can relate to you and you can set them off on a journey.

File: src/drafts/2020-04-git.md (hunk 4 above)
Comment (alxmrs) [reply src/drafts/2020-04-git.md:15]: sobering thought was my aim.

**Pull request merged**